src/main/java/hudson/plugins/im/bot/SetAliasCommand.java

```diff
@@ -89,7 +89,7 @@ public String getHelp() {
         public AliasCommand(BotCommand cmd, String commandName, String[] arguments) {
 			this.command = cmd;
 			this.commandName = commandName;
-			this.arguments = arguments;
+			this.arguments = Arrays.copyOf(arguments, arguments.length);
```

Comment:
Semantic change. I guess this won't hurt, but hope the current code doesn't have a corner case relying on that (OK, that wouldn't seem great).